source/common/protobuf/utility.h

```diff
@@ -262,8 +262,13 @@ class MessageUtil {
 
   template <class MessageType>
   static void loadFromYamlAndValidate(const std::string& yaml, MessageType& message,
-                                      ProtobufMessage::ValidationVisitor& validation_visitor) {
-    loadFromYaml(yaml, message, validation_visitor);
+                                      ProtobufMessage::ValidationVisitor& validation_visitor,
+                                      bool avoid_boosting = true) {
+    if (avoid_boosting) {
```

Comment:
nit: I think this if-else block can be changed to: `loadFromYaml(yaml, message, validation_visitor, !avoid_boosting);`